unity/Assets/Scripts/Unity/UnityMap.cs

```diff
@@ -32,7 +33,8 @@ private void Awake()
 
         private void Start()
         {
-            unityStack = GameObject.Find("Game").GetComponent("UnityStack") as UnityStack;
+            unityStack = GameObject.FindGameObjectWithTag("Stack").GetComponent<UnityStack>();
```

Comment:
We should probably explain the tag mechanism in the next meeting, might be usefull for other tasks aswell

Reply:
Will do.